### ISSUE TITLE
Support more youtube URLs

### ISSUE THREAD

**Issue by LoveIsGrief** [LoveIsGrief/videojs-externals#36]
- [ ] https://youtu.be/sxjC4CNG3_c
 - [ ] https://www.youtube.com/embed/VnxxEkr_dIA

Seems like there ain't more 🤔

**Comment by maratmkhitaryan** [comment]
Hi, can I try to do it?

**Comment by LoveIsGrief** [comment]
Go ahead @maratmkhitaryan 😃

**Comment by maratmkhitaryan** [comment]
I'm the noob, but I right understanding? Do I need rewrite regex in this file  videojs-externals/src/tech/Youtube.js? Do I need to change parseSrc()?

**Comment by LoveIsGrief** [comment]
You might have to rewrite the regex, but before you do, test the regex with the samples in this bug.

What you'll definitely have to change however is [`Youtube.nativeSourceHandler.canHandleSource`](https://github.com/LoveIsGrief/videojs-externals/blob/e78cf9763aed7091d29733385d07d339dac00ce8/src/tech/Youtube.js#L350) and [*youtube.spec.js*](https://github.com/LoveIsGrief/videojs-externals/blob/master/test/unit/youtube.specs.js) to include the tests.

For the tests, you'll have to:

 - find a pair of different "youtu.be" urls and a pair of "embed" urls
 - create new StringSourceTestSuiteGenerator objects for each pair
 - add those to the the testfactory object

I'll have to update the README and add a CONTRIBUTING to explain how to run the tests. Give me a day or two. Things will be clearer then if you don't know how to test your changes yet.

Basically however:

```bash
# install dependencies
npm install
# Install grunt
npm install -g grunt
# Run the tests once
grunt
# Run tests every time you make a change
grunt karma:watch
```

You can edit the exclude array in [*karma.conf.js*](https://github.com/LoveIsGrief/videojs-externals/blob/e78cf9763aed7091d29733385d07d339dac00ce8/test/karma.conf.js#L29)  to exclude tests and single out the youtube tests.

As I said, I have to update the README to reflect all that...
